x-pack/plugins/security_solution/server/lib/telemetry/helpers.ts

```diff
@@ -226,5 +229,10 @@ export const createUsageCounterLabel = (labelList: string[]): string => labelLis
  */
 export const extractEndpointPolicyConfig = (policyData: PolicyData | null) => {
   const epPolicyConfig = policyData?.inputs[0]?.config?.policy;
-  return epPolicyConfig ? epPolicyConfig : null;
+  const defaultPolicyConfig = getDefaultPolicyConfigAccordingToLicenseLevel(
+    licenseService.getLicenseInformation()
+  );
+  return epPolicyConfig != null
+    ? { value: merge(defaultPolicyConfig, epPolicyConfig.value) }
+    : { value: defaultPolicyConfig };
```

Comment:
Not 100% sure this is the right assumption to make if epPolicyConfig is null/undefined. Previously it would be an empty object. Does it make sense to default to the default policy config object or keep it as an empty object?